elcord.el

```diff
@@ -155,6 +155,17 @@ Swap this with your own function if you want a custom buffer-details message."
   :type 'boolean
   :group 'elcord)
 
+(defcustom elcord-editor-icon 'nil
+  "Icon to use for the text editor. When nil, use the editor's native icon."
+  :type '(choice (const :tag "Editor Default" nil)
+                 (const :tag "Emacs" "emacs_icon")
+                 (const :tag "Emacs (Pen)" "emacs_pen_icon")
+                 (const :tag "Emacs (Material)" "emacs_material_icon")
```

Comment:
Where is this 'emacs_material_icon' coming from? ~~I only have emacs_legacy_icon and emacs_pen_icon~~

**Edit**: Sorry, I didn't realize it was included in the original set